linux-anvil-aarch64/Dockerfile

```diff
@@ -0,0 +1,79 @@
+FROM continuumio/miniconda3 as builder
+
+RUN conda install constructor
+
+ADD constructors /opt/constructors
+
+RUN cd /opt/constructors/aarch64 && \ 
+    constructor --platform=linux-aarch64 .
+
+FROM arm64v8/centos:7
```

Comment:
@jjhelmus is this fine for use with your bootstrap or should this be a modern debian-like

Reply:
Is this true for c4aarch64 channel?

Reply:
yes